gvr-x3d-demo/gvr-x3d-demo/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
```

Comment:
to maintain consistency with the other apps the folder structure needs to be **gvr-x3d-demo/app**